dozer-tests/src/e2e_tests/cases/eth/dozer-config.yaml

```diff
@@ -12,7 +12,7 @@ connections:
     name: eth_logs
 sql: |
   -- Eth stats table
-  select block_number, sum(id) into eth_stats
+  select block_number, sum(id), py_udf('add', id) into eth_stats
```

Comment:
Can we have a syntax `py:add()`

Reply:
No problem

Reply:
`:` or `::` will be handled incorrectly by sqlparser and return unintended results. If we still want to use `:`, I'll check sqlparser deeply